src/components/info-button/info-button.jsx

```diff
@@ -0,0 +1,46 @@
+const bindAll = require('lodash.bindall');
+const PropTypes = require('prop-types');
+const React = require('react');
+
+require('./info-button.scss');
+
+class InfoButton extends React.Component {
```

Comment:
I think there's enough testing groundwork in place for this to have a test and for it not to take long to write it! (Still working on a Formik example), but https://github.com/LLK/scratch-www/blob/develop/test/unit/components/next-step-button.test.jsx and https://github.com/LLK/scratch-www/pull/3216/files should have enough info in them. Happy to pair on it if that'd be helpful.

Reply:
Added! Take a look!